workbench-core/swb-common-ui/src/common/labels.tsx

```diff
@@ -64,8 +64,8 @@ export const headerLabels: any = {
   signout: 'Sign out'
 };
```

Comment:
```suggestion
} as const;
```

Reply:
only if this object shouldnt be modified at all. Otherwise ignore

Reply:
same for the object below

Reply:
That doesn't work
```[eslint] Warning: src/common/labels.tsx:56:14 - (@rushstack/typedef-var) Expected a type annotation.
[eslint] Warning: src/common/labels.tsx:66:14 - (@rushstack/typedef-var) Expected a type annotation.
Encountered 2 warnings:
  [eslint] src/common/labels.tsx:56:14 - (@rushstack/typedef-var) Expected a type annotation.
  [eslint] src/common/labels.tsx:66:14 - (@rushstack/typedef-var) Expected a type annotation.
"@aws/workbench-core-swb-common-ui" completed with warnings in 11.97 seconds.
```

Reply:
its not for the lint warning, its to prevent the values within the object from being changed.